src/Controller/GameController.php

```diff
@@ -2,79 +2,101 @@
 
 namespace App\Controller;
 
-use App\Entity\Board;
+use App\Entity\Game;
 use App\Services\GameService;
-use Psr\Log\LoggerInterface;
+use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Cookie;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
 class GameController extends AbstractController
 {
     const COOKIE_KEY = 'tipsy-game';
     private $gameService;
-    private $session;
 
-    public function __construct(GameService $gameService, SessionInterface $session, LoggerInterface $logger)
+    public function __construct(GameService $gameService)
     {
         $this->gameService = $gameService;
-        $this->session = $session;
     }
 
     public function new()
     {
-        $game = $this->gameService->newGame();
+
         $playerHash = $this->generatePlayerHash();
-        $response = $this->redirectToRoute('game');
+        $game = $this->gameService->newGame($playerHash);
+        $entityManager = $this->getDoctrine()->getManager();
```

Comment:
Since you retrieve the entityManager in a lot of methods wouldn't it be better to initialize it once and for all into the contructor?